### PR TITLE
Linux Support

### PR DESCRIPTION
**Pull request by c0b41**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "uid2": "0.0.3"
   },
   "devDependencies": {
+    "electron-installer-debian": "^0.3.0",
```
Comment (rauchg) [inline package.json]: Can you pin please?

File: index.js [hunk 2]
```diff
@@ -78,7 +78,7 @@ app.on('ready', () => {
       if (fn) fn(win);
 
       // auto updates
-      if (!isDev) {
+      if (!isDev && process.platform !== 'linux') {
```
Comment (rauchg) [inline index.js]: Is there a way we can have the auto updater at least *notify* the user?

**Comment by rauchg** [comment]
Thank you so much! Pretty straightforward!

**Comment by rauchg** [comment]
By the way, can `./scripts/package.sh` generate linux binaries as well? It'd be great to incorporate that

File: index.js (hunk 2 above)
Comment (c0b41) [inline index.js]: idk, but not need linux [autoupdater](http://electron.atom.io/docs/api/auto-updater/#linux)

File: package.json (hunk 1 above)
Comment (c0b41) [inline package.json]: Fixed

**Comment by avindra** [comment]
This isn't really Linux support, this is debian-based Linux support. Generic linux binaries or an rpm package would be suitable to a wider range of distributions.

E.g., here is Atom's packaged rpm file for their 1.8.0 release:

https://github.com/atom/atom/releases/download/v1.8.0/atom.x86_64.rpm

**Comment by rauchg** [comment]
We already build `deb` and `AppImage`:
https://github.com/zeit/hyperterm/releases/tag/v0.7.1

**Comment by avindra** [comment]
I wasn't aware of the `AppImage` format, works fine. Thanks :+1:

**Pull request merged**